packages/docs/pages/pill-tabs.tsx

```diff
@@ -1,110 +1,134 @@
+              </Flex>
+            );
+            const items = [
+              { title: 'Shipping', id: 'shipping' },
+              { title: 'Orders', id: 'orders' },
+            ];
+            const onPillClick = (pillId: string) => {
+              const isPillActive = !activePills.includes(pillId);
+              const updatedPills = isPillActive
+                ? [...activePills, pillId]
+                : activePills.filter((activePillId) => activePillId !== pillId);
+
+              setActivePills(updatedPills);
+            };
+            const cards = [
+              {
+                name: 'Shipping App Pro',
+                description: 'All your shipping needs in a one stop shop.',
+                type: 'shipping',
+              },
+              {
+                name: 'Order Tracker Deluxe',
+                description: 'Track your orders across all your devices.',
+                type: 'orders',
+              },
+              {
+                name: 'Expedited Shipper',
+                description: 'The best rush rates in the country.',
+                type: 'shipping',
+              },
+              {
+                name: 'Inventory Wizard',
+                description: 'Inventory tracking app to cover all your needs.',
+                type: 'other',
+              },
+            ];
+            const isFiltered = Boolean(activePills.length);
+            const filteredCards = cards.filter((card) => activePills.includes(card.type));
+            const appCards = isFiltered ? filteredCards : cards;
+
+            return (
+              <>
+                <PillTabs activePills={activePills} items={items} onPillClick={onPillClick} />
+                <Flex>
+                  {appCards.map(({ name, description }) => (
+                    <Card key={name} name={name} description={description} />
+                  ))}
+                </Flex>
+              </>
+            );
+          }}
+          {/* jsx-to-string:end */}
+        </CodePreview>
+      </Panel>
+
+      <Panel header="Props" headerId="props">
+        <ContentRoutingTabs
+          id="props"
+          routes={[
+            {
+              id: 'pill-tabs',
+              title: 'PillTabs',
+              render: () => <PillTabsPropTable renderPanel={false} />,
+            },
+            {
+              id: 'pill-tab-item',
+              title: 'PillTabItem',
+              render: () => <PillTabItemPropTable id="pill-tabs-items-prop-table" renderPanel={false} />,
+            },
+          ]}
+        />
+      </Panel>
+
+      <Panel header="Do's and Don'ts" headerId="guidelines">
+        <GuidelinesTable
+          recommended={[
+            <>Use on pages that have a large amount of content.</>,
+            <>Be concise on the navigation labels, ideally one or two words rather than a phrase.</>,
+            <>
+              Default page view should have no <Code primary>PillTabs</Code> selected.
```

Comment:
Not sure if we should update this sentence (line 120) like this:
`When filtering a collection of items, the default page view should have no <Code primary>PillTabs</Code> selected`

Reply:
To me this change reads as if we have already clicked a pill tab (thus filtered).

Reply:
makes sense, Initially I thought that pill tabs were used as tabs, and not only filters. Maybe we could leave it just as it is.